### PR TITLE
docs(watch): update docs to the correct param strings

### PR DESCRIPTION
**Pull request by jlord**
This documentation change updates the docs for what the `fullDocument` option in `collection.watch()` takes: either `default` or `updateLookup`. These are the correct strings and match our [ChangeStreams examples](https://github.com/mongodb/node-mongodb-native/blob/3.0.0/test/functional/operation_changestream_example_tests.js#L128). 

Fixes [NODE-1272](https://jira.mongodb.org/browse/NODE-1272)

### CODE REVIEW

File: lib/collection.js [hunk 1]
```diff
@@ -2526,7 +2526,7 @@ define.classMethod('aggregate', { callback: true, promise: false });
  * @since 3.0.0
  * @param {Array} [pipeline=null] An array of {@link https://docs.mongodb.com/manual/reference/operator/aggregation-pipeline/|aggregation pipeline stages} through which to pass change stream documents. This allows for filtering (using $match) and manipulating the change stream documents.
  * @param {object} [options=null] Optional settings
- * @param {string} [options.fullDocument=none] Allowed values: ‘none’, ‘lookup’. When set to ‘lookup’, the change stream will include both a delta describing the changes to the document, as well as a copy of the entire document that was changed from some time after the change occurred.
+ * @param {string} [options.fullDocument=default] Allowed values: ‘default’, ‘updateLookup’. When set to ‘updateLookup’, the change stream will include both a delta describing the changes to the document, as well as a copy of the entire document that was changed from some time after the change occurred.
```
Comment (mbroadst) [inline lib/collection.js]: `options.fullDocument="default"`, `default` is not a type

**Comment by dandv** [comment]
The link to examples has broken, maybe https://github.com/mongodb/node-mongodb-native/blob/master/test/examples/change_streams.js is correct now?

**Comment by daprahamian** [comment]
@dandv which link are you referring to?

**Comment by dandv** [comment]
@daprahamian: the link from the OP, under "ChangeStreams examples", https://github.com/mongodb/node-mongodb-native/blob/3.0.0/test/functional/operation_changestream_example_tests.js#L128

**Comment by daprahamian** [comment]
@dandv Yes, that link is out of date, and https://github.com/mongodb/node-mongodb-native/blob/master/test/examples/change_streams.js is correct now.

**Pull request merged**